aikau/src/main/resources/alfresco/editors/TinyMCE.js

```diff
@@ -209,6 +243,64 @@ define(["dojo/_base/declare",
          } else {
             this._delayedInitConfig = config;
          }
+
+         if (this.autoResize)
+         {
+            this.alfSetupResizeSubscriptions(this.onResize, this);
+         }
+      },
+
+      /**
+       * When [autoResize]{@link module:alfresco/editors/TinyMCE} is configured to true this will
+       * respond to resize events by finding the first ancestor with height and width dimensions
+       * and then increasing the size of the TinyMCE editor to fill the available space as best
+       * it can.
+       * 
+       * @instance
+       * @since 1.0.47
+       */
+      onResize: function alfresco_editors_TinyMCE__onResize() {
+         if (this.autoResize)
+         {
+            if (this._editorInitialized)
+            {
+               // Work our way back up through the current node ancestors to find
+               // the first ancestor that have height and width set on them
+               var height, width;
```

Comment:
Also, it's very bad to rely on the string formatting of style elements returned by the browser (I've never seen that documented anywhere) by doing an indexOf with the style attribute value.

Finally, you ask if there's a width value set on the inline-style of an element and then calculate the height of the element rather than using it, which could potentially give a different value ... is this intentional? If so, then it needs to be documented as it's not logical.